lxd/instance/drivers/driver_common.go

```diff
@@ -1091,10 +1093,12 @@ func (d *common) canMigrate(inst instance.Instance) (bool, bool) {
 	for deviceName, rawConfig := range d.ExpandedDevices() {
 		dev, err := device.New(inst, d.state, deviceName, rawConfig, volatileGet, volatileSet)
 		if err != nil {
+			logger.Warnf("Instance %q cannot be auto migrated. Device %q stopped the migration due to an error: %v", inst.Name(), dev.Name(), err)
```

Comment:
We dont use the `logger.*f()` functions and should instead use contextual logging, so something like:

```go
logger.Warn("Instance will not be migrated during evacuation due to a device error", logger.Ctx{"project": inst.Project().Name, "instance": inst.Name(), "device": dev.Name(), "err": err } )
```

Reply:
Thanks, will fix that.